src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -79,6 +76,7 @@ def serialize(self, obj, attrs, user):
             "triggers": attrs.get("triggers", []),
             "projects": sorted(attrs.get("projects", [])),
             "includeAllProjects": obj.include_all_projects,
+            "owner": obj.owner.get_actor_identifier() if obj.owner else None,
```

Comment:
I actually realised we can't really use `attach_foreignkey`, because `get_actor_identifier` also makes a separate query. So we should probably entirely determine `owner` in `getattrs`, then just access it via `attrs` here.

Reply:
Ah right. Will do.